### DISCUSSION TITLE
simulate a flight mode

### DISCUSSION THREAD

**Discussion by memy009** [JSBSim-Team/jsbsim#937] [Q&A]
Hi everyone,

I have a question on how to simulate a flight mode  for an eVTOL (vertical flight includind takeOff and hover, transition between vertical and horizental, horizental flight). 

The xml I have to document models just the transition part but I find it odd in the sense that, for me, the xml model should not be linked to the flight mode. I found a video on the internet which simulates all the modes of an F35-B using the same model (https://www.youtube.com/watch?v=mCPbeqPcAH0&t=135s), all modes have been defined directly on the python interface.

So my question is:  What is the most appropriate to do, should the mode be done on the xml file or do we have the choice? 

Thank U in advance.

**Comment by seanmcleod** [comment]
There often isn't a single **correct** way of doing things with a tool like JSBSim. So difficult for us to prescribe a particular way of doing things for a project, especially when we don't have any info or insight of the project. You will need to ask your team members for their reasons for the choices they made.

If they claim that XYZ can't be done in JSBSim we can check and confirm whether that's accurate or not at the moment etc.

**Comment by memy009** [comment]
Ah Ok thank you. The one who made the xml is not giving more information but I'll try to look for the reason ...
